Makefile

```
@@ -379,6 +379,23 @@ test/addons/.docbuildstamp: $(DOCBUILDSTAMP_PREREQS) tools/doc/node_modules
 		[ $$? -eq 0 ] && touch $@; \
 	fi
 
+# All files that will be included in headers tarball should be listed as deps
+# for generating headers. The list is manually synchronized with install.py.
+ADDONS_HEADERS_PREREQS := tools/install.py \
+	config.gypi common.gypi \
+	deps/zlib/zconf.h deps/zlib/zlib.h \
+	deps/uv/include/*.h deps/v8/include/*.h \
```

I think this should be

```suggestion
	$(wildcard deps/uv/include/*.h) \
	$(wildcard deps/uv/include/*/*.h) \
	$(wildcard deps/v8/include/*.h) \
	$(wildcard deps/v8/include/*/*.h) \
	$(wildcard deps/openssl/openssl/include/openssl/*.h) \
	$(wildcard deps/openssl/config/*.h) \
```

(There are also headers from `deps/openssl/config/arch` but I think it's very rare for them to change without any of other openssl headers changing and they are hardly ever (or never?) used by the addon tests so I'd give it a pass. Also we are trying to figure out how not to bundle them unconditionally for all platforms anyway).